test/Microsoft.Azure.SignalR.Serverless.Protocols.Tests/MessagePackParseTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Buffers;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.Azure.SignalR.Serverless.Protocols.Tests
+{
+    public class MessagePackParseTests
+    {
+        public static IEnumerable<object[]> GetParameters()
+        {
+            yield return new object[] {null, Guid.NewGuid().ToString(), new object[0]};
+            yield return new object[] {Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), new object[0]};
+            yield return new object[]
+            {
+                Guid.NewGuid().ToString(), Guid.NewGuid().ToString(),
+                new object[] {Guid.NewGuid().ToString(), Guid.NewGuid().ToString()}
+            };
+            yield return new object[]
+            {
+                Guid.NewGuid().ToString(), Guid.NewGuid().ToString(),
+                new object[] {new object[] {Guid.NewGuid().ToString()}, Guid.NewGuid().ToString()}
+            };
+            yield return new object[]
+            {
+                Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), new object[] { new Dictionary<string, string>
+                {
+                    [Guid.NewGuid().ToString()] = Guid.NewGuid().ToString(),
+                    [Guid.NewGuid().ToString()] = Guid.NewGuid().ToString(),
+                }}
+            };
+            yield return new object[]
+            {
+                Guid.NewGuid().ToString(), Guid.NewGuid().ToString(),
+                new object[] {new object[] { null, Guid.NewGuid().ToString() }}
+            };
+        }
+
+        [Theory]
+        [MemberData(nameof(GetParameters))]
+        public void MessagePackParseTest(string invocationId, string target, object[] arguments)
```

Comment:
>MessagePackParseTest [](start = 20, length = 20)

add test for OpenConnectionMessage and CloseConnectionMessage as well? And JsonParseTests?

Reply:
Added.